.github/workflows/ci_linux.yml

```diff
@@ -70,3 +67,35 @@ jobs:
       with:
         name: MuseScore_${{ github.run_id }}
         path: ./build.artifacts/
+
+  run_mtests:
+    runs-on: ubuntu-16.04
+    env:
+        # Enable AddressSanitizer in the mtest build
+        CFLAGS: "-fsanitize=address -fno-omit-frame-pointer"
+        CXXFLAGS: "-fsanitize=address -fno-omit-frame-pointer"
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+    - name: Clone repository
+      uses: actions/checkout@v2
+    - name: "Configure workflow"
+      run: |
+        sudo bash ./build/ci/tools/make_build_mode_env.sh -e ${{ github.event_name }} -m ${{ github.event.inputs.build_mode }}
+        BUILD_MODE=$(cat ./build.artifacts/env/build_mode.env)
+        echo "BUILD_MODE=$BUILD_MODE" >> $GITHUB_ENV
+        DO_PUBLISH='false'
+        echo "DO_PUBLISH=$DO_PUBLISH" >> $GITHUB_ENV
+        echo "DO_PUBLISH: $DO_PUBLISH"
+    - name: Setup environment
+      run: |
+        sudo bash ./build/ci/linux/setup.sh
+    - name: Build
+      run: |
+        T_ID=${{ secrets.TELEMETRY_TRACK_ID }}; if [ -z "$T_ID" ]; then T_ID="''"; fi
+        sudo bash ./build/ci/linux/build.sh -n "${{ github.run_id }}" --telemetry "$T_ID" --buildtype mtests
```

Comment:
I think better add the option "build_mode" to `build.sh` or set "build_mode" by `buildtype` and remove "Configure workflow" step

Reply:
In principle build_mode just translates to "dev", "testing", "release", if I understood correctly. Each of them can be compiled as a debug build for tests. So in principle the "type" of build and the "mode" of build are two different things.
Or maybe I misunderstood what you are saying :sweat_smile:

Reply:
Correct, build mode - this is just the set config ("dev", "testing", "release"). For tests needs only "dev"

I think better to avoid "Configure workflow" to make it easy.

I see variants: 
1. Add --build_mode option to build.sh script and set it explicit
2. Set BUILD_MODE inside build.sh script used passed buildtype

And a new variant:
remove option `--buildtype`
add option `--build_mode`
set `--build_mode mtest`
and use `BUILD_MODE == mtest` for configure build 

that is, in fact, you can expand the values ​​of BUILD_MODE :)

Reply:
Ah, ok. :+1: 
I will also amend the "install" step (now it is failing because the environment is not set outside of the build script).